test/parallel/test-fs-promises.js

```diff
@@ -118,7 +119,9 @@ function verifyStatObject(stat) {
 
     if (common.canCreateSymLink()) {
       const newLink = path.resolve(tmpDir, 'baz3.js');
+      const newMode = 0o666;
       await symlink(newPath, newLink);
+      await lchmod(newLink, newMode);
```

Comment:
`lchmod` is `Only available on macOS`: https://nodejs.org/dist/latest-v10.x/docs/api/fs.html#fs_fs_lchmod_path_mode_callback

Reply:
Fixed it. Thank you for your review.